docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -28,6 +28,12 @@ const appList = [
   //   link: '',
   // },
   {
+    title: 'DevSketch',
```

Comment:
For now, we add them chronologically. Later we might consider doing a featured list.
Please move it to the end of the list.